Frontend/src/app/components/data-lab/active-indicator-card/active-indicator-card.component.scss

```
@@ -0,0 +1,175 @@
+  line-height: 1.4;
+  border: 1px solid transparent;
+
+  code {
+    font-family: var(--font-mono);
+    font-size: var(--fs-xs);
+  }
+}
+
+.aic-chip-category {
+  font-weight: var(--fw-semi);
+  // bg + color set inline from CATEGORY_META var() refs
```

Stylelint (17.9.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)